migrations/versions/474672feeb0b_first_init.py

```diff
@@ -1,16 +1,16 @@
-"""initial migration
+"""first init
```

Comment:
I think these migration files should be removed. It does not seem to help. At least not right now.